ComposeAdvanced/app/src/main/AndroidManifest.xml

```diff
@@ -52,6 +51,12 @@
             </intent-filter>
         </activity>
 
+        <activity
+            android:name=".presentation.ui.map.MapActivity"
+            android:exported="true"
```

Comment:
This doesn't probably need to be exported, if it's only being launched from within the app.

Reply:
Is there any clean navigation way to launch an intent? I couldn't find one and I think intent in the deeplinkoptions is just a naming option to a route.

Reply:
Does https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:navigation/navigation-runtime/src/main/java/androidx/navigation/ActivityNavigatorDestinationBuilder.kt;l=49;drc=dcb2c081d6802481afd60131f935bbcb9b0663dc work in Compose? I haven't actually tried.